strawberryfields/apps/graph/resize.py

```diff
@@ -308,6 +308,18 @@ def clique_swap(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
     supported. Degree-based node selection involves picking the node with the greatest degree,
     with ties settled by uniform random choice.
 
+    Example usage:
+
+    .. code-block::
+
+        >>> from strawberryfields.apps.graph import resize
+        >>> import networkx as nx
+        >>> graph = nx.wheel_graph(5)
+        >>> graph.remove_edge(0, 4)
+        >>> subgraph = [0, 1, 2]  # these nodes form a clique
```

Comment:
What about renaming from `subgraph` to `clique` (here and in other snippets)? `subgraph` makes it clear that it's a subgraph... but `clique` matches the argument for this function.

I have used `clique` in https://github.com/XanaduAI/strawberryfields/pull/178 so reminder to self to make sure things are consistent.

Reply:
Yes, I like it! It would also mean the comment can probably be removed

Reply:
Done!